File: Week_1/Mario/Mario_More/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //declaring variables
    int i = 0, j = 0, k = 0, t = 0;
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

// Printing pyramid
    while (i < height)
    {
        j = -1;
        t = -1;
        k = i;
        //right aligning a pyramid
        while (k < height - 1)
        {
            printf(" ");
            k++;
        }
        //printing hashes
        while (j < i)
        {
            printf("#");
            j++;
        }
        printf("  ");

        //added second block
        while (t < i)
        {
            printf("#");
            t++;
        }
        printf("\n");
        i++;
    }
}

File: Week_6/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if (height > 0 and height < 9):
        break
i = 0
j = 0
k = 0
t = 0
while (i < height):
    j = -1
    t = -1
    k = i
    while (k < height - 1):
        print(" ", end='')
        k += 1
    while (j < i):
        print("#", end='')
        j += 1
    print("  ", end='')
    while (t < i):
        print("#", end='')
        t += 1
    print("\n", end='')
    i += 1